SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$productConfiguration,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues
+	)
+	$SSOConfigTypeValues =  New-Object -TypeName psobject
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdExpInDays" -notepropertyvalue "Default" 
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdHistoryRestriction" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMinimumLength" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMaximumLength" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMinUppercase" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMinLowercase" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMinAlphabetic" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMinNumeric" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMinSpecial" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMaxConsecutiveIdenticalChar" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdMaxFailAttempts" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdAttemptsIntervalInSec" -notepropertyvalue "Default"
+	$SSOConfigTypeValues | Add-Member -notepropertyname "passwdUnlockIntervalInSec" -notepropertyvalue "Default"
+	foreach ($configuration in $productConfiguration.SSO.PSObject.Properties) {
+		if ($SSOConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			$SSOConfigTypeValues.$($configuration.Name) = $configuration.Value
+		} else {
+			Write-LogMessage -Type ERROR -Message "parameter ("$configuration.Name") not found (SSO)" -Colour Red
+		}
+	}
+	foreach ($configuration in $productConfiguration.Default.PSObject.Properties) {
+		if ($SSOConfigTypeValues.psobject.properties.match($configuration.Name).Count) {
+			if ($SSOConfigTypeValues.$($configuration.Name) -eq "Default") {
+				$SSOConfigTypeValues.$($configuration.Name) = $configuration.Value
+			}
+		}
+	}
+	$standardConfigValue = New-Object -TypeName psobject
+	$standardConfigValue | Add-Member -notepropertyname "productType" -notepropertyvalue "SSO"
+	$standardConfigValue | Add-Member -notepropertyname "productAttributes" -notepropertyvalue $SSOConfigTypeValues
+	$ppmStandardConfigValues.Add($standardConfigValue)
+}
+Function parseProductTypeNSXMgrConfiguration {
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$productConfiguration,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues
+	)
```

Comment:
Add blank line after Param block

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175